sources/packages/forms/src/form-definitions/aestdashboard.json

```diff
@@ -1,784 +0,0 @@
-{
```

Comment:
Just for understanding. It has been decided to not use form.io for AEST dashboard right?